wpk/generate_wpk_package.sh

```diff
@@ -101,20 +101,30 @@ function main() {
   do
       case "$1" in
       "-t"|"--target-system")
-          if [[ -n "$2" ]] && [[ "$2" == "linux" || "$2" == "windows" ]]; then
-            local TARGET="$2"
-            local HAVE_TARGET=true
-            shift 2
+          if [[ -n "$2" ]]; then
+            if [[ "$2" == "linux" || "$2" == "windows" ]]; then
+                local TARGET="$2"
+                local HAVE_TARGET=true
+                shift 2
+            else
+                echo "Target system must be linux or windows"
+                help 1
+            fi
           else
             echo "ERROR: Missing target system."
             help 1
           fi
           ;;
       "-b"|"--branch")
           if [[ -n "$2" ]]; then
-              local BRANCH="$(echo $2 | cut -d'/' -f2)"
-              local HAVE_BRANCH=true
-              shift 2
+            if [[ `curl https://api.github.com/repos/wazuh/wazuh-api/branches` =~ "$2" ]] || [[ `curl https://api.github.com/repos/wazuh/wazuh-api/tags` =~ "$2" ]]; then
```

Comment:
This check isn't necessary. `git` automatically checks this while cloning the branch.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20